oauth2client/contrib/metadata.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Do you think this module deserves to be a public module? Do we expect any users of oauth2client to use this directly? I would prefer for now that we rename it to `_metadata` and later promote it to public if/when the need arises.